### ISSUE TITLE
Document required vSphere permissions

### ISSUE THREAD

**Issue by albers** [hamburger-software/ansible-role-vmware_ubuntu_cloud_image#1]
Although I was able to switch the vShere GUI to English, the permissions still got translated.
So these are the permissions required to create an instance with adjusted memory and CPU count.

```
Datenspeicher
    Speicher zuteilen

Netzwerk
    Netzwerk zuweisen

Ressourcen
    Virtuelle Maschine zu Ressourcenpool zuweisen

vApp
    Import

Virtuelle Maschine
    Bestandsliste bearbeiten
        Entfernen
    Interaktion
        Ausschalten
        Einschalten
    Konfiguration ändern
        CPU-Anzahl ändern
        Neue Festplatte hinzufügen
        Speicher ändern
```

Any help in getting to the English names of these permissions would be appreciated.

**Comment by albers** [comment]
https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.security.doc/GUID-FEAB5DF5-F7A2-412D-BF3D-7420A355AE8F.html
```
DataStore
    Allocate Space

Network
    Assign Network

Resource
    Assign Virtual Machine to Resource Pool

vApp
    Import

Virtual Machine
    Inventory
        Remove
    Interaction
        Power Off
        Power On
    Configuration
        Change CPU count
        Add New Disk
        Memory
```

**Comment by albers** [comment]
resolved in d3445a38e38889a929864185e957bbf2b793fa32
